src/translators/translationOS.js

```diff
@@ -104,6 +104,7 @@ export class TranslationOS {
                 'user-agent': 'l10n.monster/TOS v0.1',
             }
             this.serviceType = serviceType ?? 'premium',
+            this.costAttributionLabel = costAttributionLabel ?? 'product',
```

Comment:
I think if it's not provided, no `cost_attribution_label` should be set rather than forcing an arbitrary label.

Reply:
ok.  i'll make the change to v2, and skip this.